### ISSUE TITLE
EnvAccessKeyNotFound error from copilot init 

### ISSUE THREAD

**Issue by erno** [aws/copilot-cli#1151]
Here's a paste of what I saw in the terminal:
```
$ copilot init
Note: It's best to run this command in the root of your Git repository.
Welcome to the Copilot CLI! We're going to walk you through some questions
to help you get set up with an application on ECS. An application is a collection of
containerized services that operate together.

Your workspace is registered to application coptest1.


Service type: Backend Service
Service name: frobber
Dockerfile: ./Dockerfile
Ok great, we'll set up a Backend Service named frobber in application coptest1 listening on port 80.

✔ Created the infrastructure to manage services under application coptest1.

✔ Manifest file for service frobber already exists at copilot/frobber/manifest.yml, skipping writing it.
Your manifest contains configurations like your container size and port (:80).

✔ Created ECR repositories for service frobber.

All right, you're all set for local development.
Deploy: Yes


✔ Created the infrastructure for the test environment.
- Virtual private cloud on 2 availability zones to hold your services     [Complete]
- Virtual private cloud on 2 availability zones to hold your services     [Complete]
  - Internet gateway to connect the network to the internet               [Complete]
  - Public subnets for internet facing services                           [Complete]
  - Private subnets for services that can't be reached from the internet  [Complete]
  - Routing tables for services to talk with each other                   [Complete]
- ECS Cluster to hold your services                                       [Complete]
- Application load balancer to distribute traffic                         [Complete]
✔ Linked account <myaccid> and region eu-west-3 to application coptest1.

✔ Created environment test in region eu-west-3 under application coptest1.
✘ get ECR repository URI: ecr describe repository coptest1/frobber: NoCredentialProviders: no valid providers in chain
caused by: EnvAccessKeyNotFound: failed to find credentials in the environment.
SharedCredsLoad: failed to load profile, <myprofile>.
EC2RoleRequestError: no EC2 instance role found
caused by: RequestError: send request failed
caused by: Get http://169.254.169.254/latest/meta-data/iam/security-credentials/: dial tcp 169.254.169.254:80: connect: no route to host
```

I'm using temporary credentials defined in .aws/config / .aws/credentials and have the right profile set in my AWS_PROFILE env var - awscli aws ecr describe-repositories will show me my frobber repository.

edit: I was using the latest release (0.1.0).

**Comment by efekarakus** [comment]
Hi @erno!

That's very strange, I wonder if this is a transient issue or if it happens everytime you run `copilot init`. Would you mind trying the command again? 
Getting the ECR repo URI uses the same session as creating the "coptest1" application.

**Comment by chrisLeeTW** [comment]
I've also faced same problem with aws-vault (https://github.com/99designs/aws-vault)

**Comment by efekarakus** [comment]
Heya! We fixed this issue recently in https://github.com/aws/copilot-cli/pull/1205 this was because during `svc deploy` we wouldn't read the `~/.aws/config` file and result in this error. It should be out with the next release!

**Comment by efekarakus** [comment]
The fixed was released in [v0.3.0](https://github.com/aws/copilot-cli/releases/tag/v0.3.0)!
